cm-gateway/cm-config/main.tf

```diff
@@ -91,7 +98,54 @@ resource "kubernetes_manifest" "gateway-issuer" {
     }
     "spec" = {
       "ca" = {
-        "secretName" = kubernetes_manifest.selfsigned-ca-cert.manifest.spec.secretName
+        "secretName" = kubernetes_manifest.selfsigned-ca-cert[0].manifest.spec.secretName
+      }
+    }
+  }
+
+  wait {
+    condition {
+      type   = "Ready"
+      status = "True"
+    }
+  }
+
+  timeouts {
+    create = "1m"
+    update = "1m"
+    delete = "30s"
+  }
+}
+
+# Create issuer with ACME type
+resource "kubernetes_manifest" "gateway-issuer-acme" {
+  count = var.issuer_type == "ACME" ? 1 : 0
+
+  manifest = {
+    "apiVersion" = "cert-manager.io/v1"
+    "kind"       = "Issuer"
+    "metadata" = {
+      "name"      = var.issuer_name
+      "namespace" = kubernetes_namespace_v1.gateway-demo.metadata[0].name
+    }
+    "spec" = {
+      "acme" = {
+        "email" : var.acme_email
```

Comment:
I guess the problem is the configuration of `acme`, could you please help me to deploy it alone to debug it?

Reply:
Ahh yes you're right there is a problem with the example email
```
status:
  acme: {}
  conditions:
  - lastTransitionTime: "2022-08-24T15:12:58Z"
    message: 'Failed to register ACME account: 400 urn:ietf:params:acme:error:invalidEmail:
      Error creating new account :: invalid contact domain. Contact emails @example.com
      are forbidden'
    observedGeneration: 1
    reason: ErrRegisterACMEAccount
    status: "False"
    type: Read
```

Reply:
... however I see that even with the correct email when the issuer does become ready it nevertheless crashes.

We should investigate for a bit more, if it doesn't help, we can try to temporarily remove the wait condition.

Reply:
> I see that even with the correct email when the issuer does become ready it nevertheless crashes.

Is it the reason that is timeout? The current creation time is only 1 minute.

https://github.com/cert-manager/testing-addons/blob/8322d6a6fc0c176be33daea175b2e5a1a285feb6/cm-gateway/cm-config/main.tf#L43